video_xblock/backends/brightcove.py

```diff
@@ -612,6 +612,5 @@ def download_default_transcript(self, url=None, language_code=None):  # pylint:
         if url is None:
             raise VideoXBlockException(_('`url` parameter is required.'))
         data = requests.get(url)
-        text = data.content.decode('utf8')
-        cleaned_captions_text = remove_escaping(text)
-        return unicode(cleaned_captions_text)
+        cleaned_captions_text = remove_escaping(data.content)
```

Comment:
Trying to test how does it work but fails:

```python
Python 3.5.2 (default, Jul 17 2020, 14:04:10)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from xml.sax.saxutils import unescape
>>> html_unescape_table = {"&amp;": "&", "&quot;": '"', "&amp;#39;": "'", "&apos;": "'", "&gt;": ">", "&lt;": "<"}
>>> import requests
>>> data = requests.get("https://google.com")
>>> unescape(data.content, html_unescape_table)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.5/xml/sax/saxutils.py", line 41, in unescape
    data = data.replace("&lt;", "<")
TypeError: a bytes-like object is required, not 'str'
```

Reply:
Oh, it's really strange because `data.content` has type `<class 'bytes'>`, just checked - it works with `str`, will decode it to string.

Reply:
done, I've added checking to the `remove_escaping` function

Reply:
@sendr 

Unfortunately, we have to re-demo this part. I need to ensure the result has a propper encoding and a user get the proper data.
And add the empty like before return statement, please.